common/flatpak-dir.c

```diff
@@ -10195,6 +10223,9 @@ flatpak_dir_find_remote_refs (FlatpakDir            *self,
   if (matched_refs == NULL)
     return NULL;
 
+  if (flags & FIND_MATCHING_REFS_FLAGS_SKIP_INSTALLED)
```

Comment:
I think it would be better to have this inside `find_matching_refs()` where the other flags are processed.